ddtrace/profiling/exporter/http.py

```diff
@@ -65,6 +68,18 @@ class PprofHTTPExporter(pprof.PprofExporter):
 
     endpoint_call_counter_span_processor = attr.ib(default=None, type=EndpointCallCounterProcessor)
 
+    def _update_git_metadata_tags(self, tags):
+        """
+        Update profiler tags with git metadata
+        """
+        gitmetadata.clean_tags(tags)
```

Comment:
can you maybe add a comment to say why we are removing tags? I think his is because `get_git_tags` will look into the DD_TAGS as well, and here `tags` exactly contains the content of `DD_TAGS`, is that the reason?